paasta_tools/tron_tools.py

```diff
@@ -697,13 +697,14 @@ def get_pool(self) -> str:
         override this value. To control this, we have an optional config item that we'll puppet onto Tron masters
         which this function will read.
         """
-        return (
-            self.config_dict.get(
+        if self.get_executor() == "spark":
+            pool = load_system_paasta_config().get_spark_driver_default_pool_override()
```

Comment:
it might be worth starting a discussion on how to allow for configuring the driver pool per-action since I assume eventually we'll want to also run GPU-using drivers through tron